packages/wallets/src/metamask/metamask.page.ts

```diff
@@ -218,21 +218,9 @@ export class MetamaskPage implements WalletPage {
     });
   }
 
-  async signTx(page: Page) {
-    await test.step('Sign TX', async () => {
-      await new WalletOperationPage(page).signTransaction();
-    });
-  }
-
-  async cancelSignTx(page: Page) {
-    await test.step('Cancel Sign TX', async () => {
-      await new WalletOperationPage(page).cancelSignTransaction();
-    });
-  }
-
   async rejectTx(page: Page) {
```

Comment:
Should we remove wording wording reject->cancel?

Reply:
renamed all locators and functions (`reject` to `cancel`)